backend/app/api/routers/cipher.py

```
@@ -74,9 +74,39 @@ async def simulate_geometry_event(cgp: Dict[str, Any] = Body(...)):
     """
     Simulate receiving a CHIT Geometry Packet (CGP).
     Decodes it and returns the A2UI SurfaceUpdate to visualize it.
+
+    Zeta frequencies are dynamically computed from:
+    1. Constellation point spectra (if available)
+    2. Point coordinate distributions
+    3. Falls back to default zeta zeros if insufficient data
     """
     decoded = chit_service.decode_cgp(cgp)
```

Ruff (0.14.11)

Local variable `decoded` is assigned to but never used

Remove assignment to unused variable `decoded`

(F841)